tests/Oryx.BuildImage.Tests/Python/PythonDynamicInstallationTest.cs

```diff
@@ -21,24 +21,20 @@ public PythonDynamicInstallationTest(ITestOutputHelper output) : base(output)
         {
         }
 
-        public static TheoryData<string> ImageNameData
+        public static IEnumerable<object[]> ImageNameData()
```

Comment:
In general strongly type objects are preferred over `object`...in this case you could have used `TheoryData<string, string>`